main.go

```diff
@@ -674,11 +679,13 @@ func _main(ctx context.Context) error {
 	// This dumps the cache charts data into a file for future use on system
 	// exit.
 	defer func() {
-		er := explorer.WriteCacheFile(dumpPath)
-		if er != nil {
-			log.Errorf("WriteCacheFile failed: %v", er)
-		} else {
-			log.Debug("Dumping the charts cache data was successful")
+		if usePG {
+			er := explorer.WriteCacheFile(dumpPath)
```

Comment:
Not useful in lite mode.

Reply:
pretty sure, but do check this assertion please, @dmigwi

Reply:
You right. You could probably include the whole defer function in the if statement.